google-cloud-pubsub/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Release History
 
+### 0.40.0 / 2019-08-27
+
+#### Features
+
+* Add ReceivedMessage#delivery_attempt
```

Comment:
so, this is slightly more complicated than it sounds at first, because the version is also reflected in the files in the PR, so even if we tagged a `0.39.2` release, as a side effect of updating the release title, we'd need some way to update the corresponding files in the PR...

I kind of prefer the idea of using footers that indicate override the release level:
 
```
feat: adds a new alpha feature

Release-As: patch
```